src/assignment/modals/AddBlockModal.tsx

```diff
@@ -65,15 +65,25 @@ const AddBlockModal: FunctionComponent<AddBlockModalProps> = ({
 					'assignment/modals/add-block___start-je-opdracht-vanaf-een-bestaande-collectie-fragmenten-uit-je-werkruimte'
 				),
 			},
-			{
-				type: AssignmentBlockType.ZOEK,
-				icon: BLOCK_ITEM_ICONS()[AssignmentBlockType.ZOEK](),
-				title: t('assignment/modals/add-block___zoeken-bouwen'),
-				description: t(
-					'assignment/modals/add-block___leer-leerlingen-zelf-bronnen-zoeken-of-laat-ze-een-collectie-samenstellen'
-				),
-				disabled: blocks.findIndex((block) => block.type === AssignmentBlockType.ZOEK) >= 0,
-			},
+			// Only show the search block option in addBlockModal when the blocks do not contain a search block yet
+			...(blocks.find(
+				(b) => b.type === AssignmentBlockType.ZOEK || b.type === AssignmentBlockType.BOUW
+			)
+				? []
+				: [
+						{
+							type: AssignmentBlockType.ZOEK as AddBlockModalType,
+							icon: BLOCK_ITEM_ICONS()[AssignmentBlockType.ZOEK](),
+							title: t('assignment/modals/add-block___zoeken-bouwen'),
+							description: t(
+								'assignment/modals/add-block___leer-leerlingen-zelf-bronnen-zoeken-of-laat-ze-een-collectie-samenstellen'
+							),
+							disabled:
```

Comment:
Is het niet tonen volgens de specificaties? Niet ook gewoon disablen als er zoek || bouw blok is? 

Nu is die disabled state niet meer nuttig hier

Reply:
goed punt. Blijkbaar ontbrak enkel de check voor een BOUW blok:
![image](https://user-images.githubusercontent.com/1710840/179196584-f605fb4b-6fe1-4a48-b7f1-0adcb55e7525.png)